gemfirexd/shared/build.gradle

```diff
@@ -32,6 +32,7 @@ task createVersionPropertiesFile(dependsOn: 'processResources') {
     def props = [
       'Product-Name'      : PRODUCT_NAME,
       'Product-Version'   : version,
+      'Enterprise-Edition': true,
```

Comment:
Shouldn't this be based on a property instead of always true like: rootProject.hasProperty('snappydata.enterprise')

For our unit tests which require some of these features, the property can be set by default in users ~/.gradle/gradle.properties